runtime/Cpp/runtime/src/atn/PredictionContext.h

```diff
@@ -65,13 +65,10 @@ namespace atn {
     /// </summary>
     const size_t cachedHashCode;
 
-  protected:
-    explicit PredictionContext(size_t cachedHashCode);
-
   public:
     virtual ~PredictionContext() = default;
 
-    virtual PredictionContextType getContextType() const = 0;
+    PredictionContextType getContextType() const;
```

Comment:
The implementation seems to be missing, resulting in a linking error.

Reply:
Woops. Fixed.